tools/c64/c64upload.py

```
@@ -0,0 +1,59 @@
+    args = parser.parse_args()
+
+    if not args.host:
+        print("Error: Missing hos.")
+        exit(1)
+
+    source_path = Path(args.source)
+    target_path = PosixPath("SD") / source_path.name
+
+    print(f"Deploying {source_path} to C64 host {args.host}...")
+
+    ftp = FTP(args.host)
```

Ruff (0.14.10)

FTP-related functions are being called. FTP is considered insecure. Use SSH/SFTP/SCP or some other encrypted protocol.

(S321)